src/components/views/create/CreateProject.tsx

```diff
@@ -389,14 +397,15 @@ const CreateProject: FC<ICreateProjectProps> = ({ project }) => {
 								disabled={isLoading}
 							/>
 						</Buttons>
-						{addressModalChainId && (
+						{addressModalChainId !== undefined && (
 							<CreateProjectAddAddressModal
 								networkId={addressModalChainId}
-								setShowModal={setAddressModalChainId}
-								userAddresses={userAddresses}
-								onSubmit={() =>
-									setAddressModalChainId(undefined)
-								}
+								chainType={addressModalChainType}
+								userAddresses={userUniqueAddresses}
+								setShowModal={() => {
+									setAddressModalChainId(undefined);
+									setAddressModalChainType(undefined);
+								}}
```

Comment:
Please revert changes are not required for solana support here.

Reply:
Done!